client.go

```
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	arguments := os.Args
+	if len(arguments) == 1 {
+		fmt.Println("Please provide host:port.")
+		return
+	}
+
+	connect := arguments[1]
+	c, err := net.Dial("tcp", connect)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	for {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print(">> ")
+		text, _ := reader.ReadString('\n')
```

Why do you ignore an error returned by `ReadString()` function ?

---

fixed 
commit: 4d48f92c1d2ee94fcf6fc8299bc3b759a0b1dc90